src/Sulu/Component/Webspace/Loader/XmlFileLoader11.php

```diff
@@ -73,4 +84,27 @@ protected function generateTemplates(Webspace $webspace)
 
         return $webspace;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function generatePortals()
```

Comment:
ah right ... i also have removed the resource-locator strategy from 1.0 so i can remove this method.